Sming/SmingCore/Delegate.h

```diff
@@ -229,11 +229,13 @@ template <class ReturnType, class... ParamsList> class Delegate<ReturnType(Param
 	void copy(const Delegate& other)
 	{
 		if(impl != other.impl) {
-			if(impl)
+			if(impl != nullptr) {
```

Comment:
Any possible side effects that you can think of from the changes in Delegate.h ?

Reply:
No side effects. Because `impl` has a static initialiser we can be certain that all the constructors get a default `nullptr` value for this, so setting it to `nullptr` again isn't necessary.